net.stabyourself.nottetris2.yml

```diff
@@ -77,7 +79,12 @@ modules:
         path: love-sdl-headers-path.patch
 
   - name: nottetris2
+    buildsystem: simple
+    build-commands:
+      - unzip nottetris2-linux.zip
+      - install -Dm644 'Not Tetris 2.love' /files/nottetris2.love
     sources:
-      - type: archive
```

Comment:
Will be solved via https://github.com/flatpak/flatpak-builder/pull/166 but it's fine as workaround for now.